tools/perfmon/run.py

```diff
@@ -40,9 +40,8 @@
 env['NAPARI_PERFMON'] = config_path
 
 subprocess.check_call(
-    'napari ' + ' '.join(args.napari_args),
+    ['python', args.example_script],
```

Comment:
This seems really silly, but I couldn't think of a different way to do this. Suggestions and improvements welcome!

Reply:
Seems fine. Given that this tool is (mostly) for development, I agree with the switch from 'napari <foo>' to 'python <foo>'.